ledger/accountdb.go

```diff
@@ -1488,3 +1608,9 @@ func (iterator *catchpointPendingHashesIterator) Close() {
 		iterator.rows = nil
 	}
 }
+
+// before compares the round numbers of two persistedAccountData and determines if the current persistedAccountData
+// happened before the other.
+func (pac *persistedAccountData) before(other *persistedAccountData) bool {
```

Comment:
I think that the `func (t Time) Before(u Time) bool` is much better match for the functionality..
Considering public/private here, I think that private makes more sense, since it's being used internally only ( and admittedly, only in one place )